docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 
 services:
-  geth: # this is Optimism's geth client
+  op-geth: # this is Optimism's geth client
```

Comment:
do we need to change this name?

Reply:
since it's a fork, wouldn't it be easier to sync if we have less changes?

Reply:
In that case I can revert these changes.

Reply:
Updated!